user/common-tasks/disposablevm.md

```diff
@@ -153,6 +153,21 @@ $ qvm-open-in-vm @dispvm:online-dvm-template https://www.qubes-os.org
 
 This will create a new DisposableVM based on `online-dvm-template`, open the default web browser in that DisposableVM, and navigate to `https://www.qubes-os.org`.
 
+#### Example of RPC policies to allow this behavior
+
+In dom0, add the following line at the beginning of the file `/etc/qubes-rpc/policy/qubes.OpenURL`
+~~~
+$anyvm @dispvm:online-dvm-template allow
```

Comment:
Please use @ consistently - it should be `@anyvm`.

Reply:
Ok, thanks : )